### DISCUSSION TITLE
Wikipedia style previews on mouse hover.

### DISCUSSION THREAD

**Discussion by jmhimara** [quarto-dev/quarto-cli#5381] [Q&A]
Is it possible to include this feature on book or website layouts? Basically, show a preview of the page whenever the mouse hovers over the link: https://en.wikipedia.org/wiki/Wikipedia:Tools/Navigation_popups

I've seen options for enabling this on reference and footnote links, but not page links. This is an example of the same thing in HTML+CSS+JS :https://stackoverflow.com/questions/23251569/preview-page-on-link-hover

**Comment by mcanouil** [comment]
See the opened feature request #2427.
I am closing this as a duplicate.
Feel free to follow up on the other thread.
